Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:10-alpine
+
+# Create app directory
+WORKDIR /usr/src/makina-api/
+
+# Bundle app source
+COPY . .
```

Comment:
But, as far as I know, we don't have a dist folder. Furthermore, it doesn't copy everything, there is a .dockerignore file, we can write there files or folders that we want to exclude.

Reply:
dist folder is generated by build task which makes use of tsc directly.

.dockerignore is a cool way though it is easier to pass in just what's needed instead of copying everything then trying to filter out what's not needed.

besides, while building an image, docker engine creates separate layers for each dockerfile command and it helps a lot on speeding up consequent builds because layers are cached and it just detects the result wouldn't change the cached layer so it uses cached one instead of rebuilding whenever it can.

for example, for a dev build let's say, we might only change test coverage which wouldn't effect the final js files and for that case copying everything would still create a new layer whereas the cache would be fine for the opposite since the contents for dist wouln't be changed.